Victor (#6)
AllisonOge/electromagnetism-and-antenna-theory-v2-1: TeX_files/chapter09.tex

\begin{center}
\textbf{PROBLEM 2}
\end{center}

> If the wave in the previous example is traveling in the negative ${x}$ direction, with all other things same. Find the instantaneous  voltage at the same time and at the same location.\\

Reviewer: ```suggestion
Given that the wave in the previous example is travelling in the negative ${x}$ direction and all other parameters are kept the same, what is the instantaneous voltage at the same time and at the same location?\\
```
